extensions/autodoc_placeholder/__init__.py

```diff
@@ -19,6 +19,8 @@ def setup(app):
     directives.register_directive('autodata', PlaceHolder)
     directives.register_directive('automethod', PlaceHolder)
     directives.register_directive('autoattribute', PlaceHolder)
+    directives.register_directive('autofield', PlaceHolder)
+    directives.register_directive('autoodooclass', PlaceHolder)
```

Comment:
Since this uses the Odoo lingo of "field" for the class attributes, why not make this `automodel` rather than the difficult to read `autoodooclass`?